cocos/core/asset-manager/asset-manager.ts

```diff
@@ -288,6 +288,7 @@ export class AssetManager {
 
     private _parsed = parsed;
     private _parsePipeline = BUILD ? null : new Pipeline('parse existing json', [this.loadPipe]);
+    private _projectBundles: string[] = [];
```

Comment:
Is this designed to unify all bundle information.

Reply:
Used in game view for now